common_utils.sh

```diff
@@ -490,7 +490,7 @@ LATEST_PP_6=$LATEST_PP_6p0
 LATEST_PP_7p0=7.0.0
 LATEST_PP_7p1=7.1.1
 LATEST_PP_7p2=7.2.0
-LATEST_PP_7p3=7.3.3
+LATEST_PP_7p3=7.3.7
```

Comment:
```suggestion
LATEST_PP_7p3=7.3.8
```
7.3.8 is now out:
https://www.pypy.org/posts/2022/02/pypy-v738-release.html